CHANGES.rst

```diff
@@ -6,7 +6,27 @@ Changes
 Unreleased
 ==========
 
-- Added the `preview-rst` helper script
+- 
+
+2.0.1 - 2022/03/02
+==================
+
+Breaking Changes
+----------------
+
+**IMPORTANT**: You must update your Sphinx project's ``Makefile`` to pick up
+these changes. See ``docs/Makefile`` for details.
+
+Features
+--------
+
+- Added the ``preview-rst`` helper script
+
+Fixes
+-----
+
+- Fixed no vale case
+- Vale ignores ``eggs`` folder
```

Comment:
This is a good thing, right?

Reply:
Ah, I see. This change from d1e87e315 wasn't reflected within the change log yet. Thanks!